packages/protocol/contracts/lib/Checkpointing.sol

```diff
@@ -38,6 +40,7 @@ library Checkpointing {
      * @param _time Point in time to register the given value
      * @param _value Numeric value to be registered at the given point in time
      */
+     // Note: do we need this to be uint256, or can we do uint192 to make it simpler?
```

Comment:
Staking contract works with `uint256` so we need this unless we pollute it with `uint192` conversions. I like it better here.

Reply:
Gotcha, let's remove the note then :)

I used a library in the voting connectors for this conversion, since the external `uint192` type felt clearer, but it's roughly the same either way.